src/modules/sync/diff/workflow.ts

```diff
@@ -16,7 +16,7 @@ const workflowStepHandler: Handler<WorkflowStepSyncModel> = makeObjectHandler({
   color: baseHandler,
   transitions_to: makeArrayHandler(
     t => t.step.codename,
-    makeObjectHandler({ step: makeLeafObjectHandler({ codename: (source, target) => source === target }) }),
+    makeObjectHandler({ step: makeLeafObjectHandler({ codename: (source: any, target: any) => source === target }) }),
```

Comment:
why is this needed? we don't do `any` here unless completely necessary

Reply:
Understand. I can't reproduce and happyly reverted.